Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1747,9 +1747,9 @@ def fetch_incidents():
         last_fetch, _ = parse_date_range(FETCH_TIME, to_timestamp=True)
 
     current_fetch = last_fetch
-
     incidents = []
-    threats = get_threats_request()
+    last_fetch_date_string = timestamp_to_datestring(last_fetch, '%Y-%m-%dT%H:%M:%S.%fZ')
+    threats = get_threats_request(created_after=last_fetch_date_string)
     for threat in threats:
         # If no fetch threat rank is provided, bring everything, else only fetch above the threshold
         if not FETCH_THREAT_RANK or (FETCH_THREAT_RANK and threat.get('rank') >= FETCH_THREAT_RANK):
```

Comment:
`FETCH_THREAT_RANK` will always a value since you set a default, so you can remove this